nodewatcher/modules/analysis/channel_allocation/migrations/0003_auto_20160822_0426.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-08-22 02:26
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('channel_allocation', '0002_remove_nodechannel_channel_width'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='nodechannel',
+            name='optimal_channel_interference',
+            field=models.FloatField(),
+        ),
+    ]
```

Comment:
I can probably edit `0001_initial.py` with subsequent changes. Will try and see.